lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -446,10 +474,61 @@ export class BrowserAuthError extends AuthError {
         return new BrowserAuthError(BrowserAuthErrorMessage.authCodeRequired.code, BrowserAuthErrorMessage.authCodeRequired.desc);
     }
 
+    /**
+     * Create an error when an authorization code or native account ID is required but not provided
+     */
+    static createAuthCodeOrNativeAccountIdRequiredError(): BrowserAuthError {
+        return new BrowserAuthError(BrowserAuthErrorMessage.authCodeOrNativeAccountRequired.code, BrowserAuthErrorMessage.authCodeOrNativeAccountRequired.desc);
+    }
+
     /**
      * Create an error when IndexedDB is unavailable
      */
     static createDatabaseUnavailableError(): BrowserAuthError {
         return new BrowserAuthError(BrowserAuthErrorMessage.databaseUnavailable.code, BrowserAuthErrorMessage.databaseUnavailable.desc);
     }
+
+    /**
+     * Create an error when native token acquisition is not possible
+     */
+    static createUnableToAcquireTokenFromNativePlatformError(): BrowserAuthError {
```

Comment:
Would it be worth making a new class of errors for WAM-bridge specific errors?

Reply:
I did consider this but felt that `NativeAuthError` should be for errors thrown by the native layer (similar to `ServerError` for server errors) and that we should try to keep our errors in the same place. Most of these are only thrown internally and wont surface to the end user. Open to revisiting that, however.